src/main/java/com/vfasad/controller/KanbanController.java

```diff
@@ -60,13 +61,15 @@ public String moveOrder(
             @RequestParam(required = false)
             @ElementMin(value = 0, message = "Quantities cannot be zero or negative.")
                     List<Double> actualQuantities,
-            @RequestParam(required = false) Long teamId) {
+            @RequestParam(required = false) Long teamId,
+            HttpServletRequest request) {
 
+        String url = request.getRequestURL().toString();
```

Comment:
Now we move order from Kanban, so we can use that url directly from request. But it's possible to perform that action from some other page, so I would propose to call method getHost() of URL object pass it to the service or pass URL object to the service itself.